modules/400-descheduler/templates/configmap.yaml

```diff
@@ -1,111 +1,128 @@
+{{- range $d := .Values.descheduler.internal.deschedulers }}
+  {{- $deschedulerSpec := $d.spec }}
+  {{- $commonParameters := $deschedulerSpec.commonParameters }}
+  {{- $strategies := $deschedulerSpec.strategies }}
 ---
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: descheduler-policy-configmap
+  name: descheduler-policy-{{ $d.metadata.name }}
   namespace: d8-descheduler
   {{- include "helm_lib_module_labels" (list .) | nindent 2 }}
 data:
   policy.yaml: |
     apiVersion: "descheduler/v1alpha1"
     kind: "DeschedulerPolicy"
+    {{- if $strategies }}
+    {{- $strategies | toYaml | nindent 4 }}
```

Comment:
Template tests are required to check what we generate for different options. I think you will find some errors after writing them.

Reply:
Written tests.